labels/de-DE.json

```diff
@@ -265,6 +265,8 @@
   "mobileOrderConfirmationMessage": "{0} Bestellbenachrichtigungen. Wir werden Ihnen eine SMS senden, wenn sich Ihr Bestellstatus ändert. Die Meldungsfrequenz kann variieren. Antworte {1}, um dich abzumelden. Es können Nachrichten- und Datentarife gelten",
   "mobileOrderOptInMessage": "Vielen Dank, dass Sie sich für SMS-Benachrichtigungen für Ihre Curbside-Bestellung angemeldet haben. Wir werden Sie benachrichtigen, wenn Ihre Bestellung zur Abholung bereit ist.",
   "mobilePickLocation": "Ramasser / Expédier de:",
+  "mobilePickupReminder":"Rappel, les articles de votre commande {0} sont prêts à être ramassés. Vérifiez la commande à {1}",
```

Comment:
This file is de-DE which means German. A lot of translations in this file are incorrectly French but they should all be in German.